### DISCUSSION TITLE
Extraction of data from a plan that cuts a 3D microstructure

### DISCUSSION THREAD

**Discussion by J-AKAZ** [neperfepx/neper#392] [Q&A] [answered]
Hello,

In order to compare Neper's 3D microstructure with my EBSD data (2D), I would like to know if there is a method on Neper that allows to extract the data on a cutting plane in the microstructure and to extract the equivalent diameters of the grains and/or their equivalent surfaces?

thank you,

**Comment by rquey** [comment, marked as answer]
Yes, use [-transform slice](https://neper.info/doc/neper_t.html#cmdoption-transform) to slice the tessellation and [-statcell](https://neper.info/doc/neper_t.html#cmdoption-statcell) to get properties of the cells on the slice.

Example:
```
neper -T -n 1000 -transform "slice(0.5,0,0,1)" -statcell diameq,area
```
